### ISSUE TITLE
[Regression][v0.16.0] sendig-dart rejected as unsupported standard despite existing in cache

### ISSUE THREAD

**Issue by Janakiraman-311** [cdisc-org/cdisc-rules-engine#1727]
### Standard

CDISC SEND

### Reference Rule ID(s)

Standard: sendig-dart (versions 1-1, 1-2) Reference Rule ID(s): N/A — General engine CLI failure

### Conformance Rule ID(s) (if published in CORE)

N/A — Fails before rule execution reaches any rule

### JIRA Ticket

https://jira.cdisc.org/projects/CORERULES/issues/CORERULES-

### CLI Command Used (if applicable)

python core.py validate -s sendig-dart -v 1-1 -d /path/to/data and python core.py validate -s sendig-dart -v 1-1 -d /path/to/data --custom-standard 

### Rule YAML

```YAML
Not applicable. The engine fails during CLI argument validation in core.py before any rule YAML is read or parsed. DART rules are confirmed present in cache — core.exe list-rule-sets correctly returns sendig-dart
```

### Attach any sample test data file(s)

Any DART study folder containing valid .xpt files (DM, BG, BW, CL) reproduces this. Data is never read — the failure occurs at argument validation.

### Attach any output report and/or log file(s)

[ERROR 2026-05-07 12:38:19,555 - core.py:608] - Standard 'sendig-dart' is not 
a supported standard. Supported standards: adam, sdtmig, sendig, tig, usdm. 
Use --custom-standard flag for custom standards.
Process finished with status: 2

### Expected output

Validation initialises successfully, loads DART rules from cache, and produces a JSON report — as it did in v0.15.0.

### Any Additional Information

Additional Information:
This is a confirmed regression introduced in v0.16.0 with no available workaround.
1. Root Cause : Strict Enum Validation
introduced StandardTypes in cdisc_rules_engine/enums/standard_types.py. The -s argument is now validated against this enum before any cache lookup occurs.
Current enum contains only 5 values:
ADAM, SDTMIG, SENDIG, TIG, USDM
Missing despite being valid CDISC Library standards with rules in cache:
SENDIG_DART

2. Impact
Users cannot validate DART studies on v0.16.0. Downgrade to v0.15.0 is the only current resolution.
